### DISCUSSION TITLE
2023-05-24 Next.js 真的适合你的项目吗 ~ 蒋继发 (ThaddeusJiang)

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#40] [Announcements]
# 2023-05-24 Next.js 真的适合你的项目吗 ~ 蒋继发 (ThaddeusJiang)

这是我在犹豫是否升级至 Next.js App Router 时的一些思考。我有很多项目，每个项目的情况各不相同，在 App Router 前，我一直使用 Next.js 作为唯一的前端框架。

https://thaddeusjiang.com/2023-05-24-next-js-zhen-de-shi-he-ni-de-xiang-mu-ma.html

**Comment by weykon** [comment]
一开始使用的时候，我觉得特别艰难，发现并不是很自由，以为是自己不会用，用错了。
原来他的局限性确确实实比较大。
谢谢分享
